save-core/src/commonNonJsMain/kotlin/org/cqfn/save/core/Save.kt

```diff
@@ -38,21 +36,23 @@ class Save(
     private val saveProperties: SaveProperties,
     private val fs: FileSystem,
 ) {
+    private val reporter = getReporter(saveProperties)
```

Comment:
Maybe, make this field internal, if it is intended only for tests in save-core? Seems weird, that performAnalysis returns reporter, which probably will be stateless in real scenarios